config/defaults_template.yaml

```diff
@@ -1,8 +1,9 @@
 # See ./defaults.go for documentation on each field.
 
-driverPool: "{{ .DriverPool }}"
-
-workerPool: "{{ .WorkerPool }}"
+defaultPoolLabels:
+  client: default-client-pool
+  driver: default-driver-pool
+  server: default-server-pool
```

Comment:
If we already have the default for driver and workers, do we still want the configuration.go file to error out when the pool is not set? See https://github.com/grpc/test-infra/blob/23ce525193460dbd1633a080e85b72cacd1e0784/config/cmd/configure.go#L104-L110

Reply:
Oops. How did that get missed? I'll add it.

Reply:
Also if "default-client-pool", "default-driver-pool" and "default-server-pool" become our keys for our NodeSelectors keys, the values corresponding to those strings would be labels of the Nodes, wondering how do we know what are the labels that the Node are having? Confused....

Reply:
Let's talk about it over gvc. 👍🏻